lib/task-data/tasks/generate-tag.js

```diff
@@ -6,8 +6,10 @@ module.exports = {
     friendlyName: 'Generate Tag',
     injectableName: 'Task.Catalog.GenerateTag',
     implementsTask: 'Task.Base.Catalog.GenerateTag',
+    schemaRef: 'generate-tag.json',
     options: {
-        nodeId: null
+        nodeId: null,
+        nodes: null
```

Comment:
If you agree to change to my proposed schemas, then both `null` options need be removed here as the `null` violates the schema.